src/rpc/server.cpp

```diff
@@ -470,6 +470,163 @@ UniValue CRPCTable::execute(const JSONRPCRequest &request) const
+            "createpsbt",
+            "createrawtransaction",
+            "decodepsbt",
+            "decoderawtransaction",
+            "decodescript",
+            "deriveaddresses",
+            "disconnectnode",
+            "dumpprivkey",
+            "dumptxoutset",
+            "dumpwallet",
+            "echo",
+            "echojson",
+            "estimaterawfee",
+            "estimatesmartfee",
+            "finalizepsbt",
+            "fundrawtransaction",
+            "generate",
+            "generateblock",
+            "generatetoaddress",
+            "generatetodescriptor",
+            "getaddednodeinfo",
+            "getaddressesbylabel",
+            "getaddressinfo",
+            "getbalance",
+            "getbalances",
+            "getbestblockhash",
+            "getblock",
+            "getblockchaininfo",
+            "getblockcount",
+            "getblockfilter",
+            "getblockhash",
+            "getblockheader",
+            "getblockstats",
+            "getblocktemplate",
+            "getchaintips",
+            "getchaintxstats",
+            "getconnectioncount",
+            "getdescriptorinfo",
+            "getdifficulty",
+            "getindexinfo",
+            "getmemoryinfo",
+            "getmempoolancestors",
+            "getmempooldescendants",
+            "getmempoolentry",
+            "getmempoolinfo",
+            "getmininginfo",
+            "getnettotals",
+            "getnetworkhashps",
+            "getnetworkinfo",
+            "getnewaddress",
+            "getnodeaddresses",
+            "getpeerinfo",
+            "getrawchangeaddress",
+            "getrawmempool",
+            "getrawtransaction",
+            "getreceivedbyaddress",
+            "getreceivedbylabel",
+            "getrpcinfo",
+            "gettransaction",
+            "gettxout",
+            "gettxoutproof",
+            "gettxoutsetinfo",
+            "getunconfirmedbalance",
+            "getwalletinfo",
+            "getzmqnotifications",
+            "help",
+            "importaddress",
+            "importdescriptors",
+            "importprunedfunds",
+            "importpubkey",
+            "importwallet",
+            "invalidateblock",
+            "joinpsbts",
+            "keypoolrefill",
+            "listaddressgroupings",
+            "listbanned",
+            "listlabels",
+            "listlockunspent",
+            "listreceivedbyaddress",
+            "listreceivedbylabel",
+            "listsinceblock",
+            "listtransactions",
+            "listunspent",
+            "listwalletdir",
+            "listwallets",
+            "loadwallet",
+            "lockunspent",
+            "logging",
+            "mockscheduler",
+            "ping",
+            "preciousblock",
+            "prioritisetransaction",
+            "pruneblockchain",
+            "psbtbumpfee",
+            "reconsiderblock",
+            "removeprunedfunds",
+            "rescanblockchain",
```

Comment:
This list seems to need an update for new RPCS (`restorewallet` comes to mind, not sure if there are others)